src/classes/CON_ContactMerge_TDTM.cls

```diff
@@ -106,23 +106,32 @@ public class CON_ContactMerge_TDTM extends TDTM_Runnable {
     }
 
     /*******************************************************************************************************
-    * @description After a Contact Merge, call this Future routine to perform assorted fixups that need to
+    * @description After a Contact Merge, call this async routine to perform assorted fixups that need to
     * occur which include:
     *   - update the Account name and greetings 
     *   - move Opps to the Master Contact's Account 
     *   - recalculate Household Soft Credit OCRs 
     *   - recalculate Opportunity Rollups
     *
-    * @param mergeKeys The list of mergeKeys that provide oldContactId, oldAccountId, and masterContactId
-    * @return null
     ********************************************************************************************************/
-    @Future
-    private static void handleContactMergeFixupsFuture(List<String> mergeKeys) {
-        if (mergeKeys == null || mergeKeys.isEmpty()) {
-            return;
+    private class AsyncContactMergeFixup implements Queueable {
```

Comment:
Suggestion: `ContactMergeFixupQueueable`?

Reply:
I actually went with async intentionally so no matter what we decided to use we could keep the name the same... thoughts? I also noticed our error class was named with Async.

Reply:
It implements Queueable, so I thought it would help to be clear and descriptive in the class name, similarly how we add `future` or `batch` in respective method/class names. It is optional.